examples/03_fit_predict_plot_midwest_survey.py

```diff
@@ -69,7 +69,7 @@
 encoder_dict = {
     'one-hot': OneHotEncoder(handle_unknown='ignore', sparse=False),
     'similarity': SimilarityEncoder(similarity='ngram'),
-    'minhash': MinHashEncoder(n_components=10, ngram_range=(2, 4),
+    'minhash': MinHashEncoder(n_components=30, ngram_range=(2, 4),
```

Comment:
Why not n_components=100 here too ?

Reply:
Well, going to 100 only improves from 0.676 to 0.678.
Both are similar to the similarity encoding score (0.674).

Reply:
ok, thanks for the info